pkg/operator/operatorclient/operatorclient.go

```diff
@@ -34,12 +36,17 @@ func (c OperatorClient) Informer() cache.SharedIndexInformer {
 	return c.Informers.Operator().V1alpha1().CertManagers().Informer()
 }
 
+// GetOperatorState returns the Operator state. If it's missing (e.g. the certmanager CR hasn't been creates),
+// it will create a default one.
 func (c OperatorClient) GetOperatorState() (*operatorv1.OperatorSpec, *operatorv1.OperatorStatus, string, error) {
```

Comment:
It seems strange to me that `GetOperator` state has side effects. Do we have a better place?

Reply:
Yeah... thinking about this a bit more it's weird. Let me introduce a new Controller to ensure it's always there. It will be cleanest solution I think.

Reply:
Done